src/packages/hex.gleam

```diff
@@ -3,41 +3,184 @@ import gleam/json
 import gleam/http
 import gleam/result
 import gleam/hackney
+import gleam/list
 import gleam/dynamic.{Decoder} as d
+import time/parse.{parse_iso8601_to_gregorian_seconds}
+import gleam/pgo
+import gleam/string
+import gleam/int
+import gleam/option.{Option}
 
 pub type Error {
   HackneyError(hackney.Error)
   JsonError(json.DecodeError)
+  DatabaseError(pgo.QueryError)
+  MiscError(String)
+  NilError(Nil)
```

Comment:
These two error messages don't convey any information, let's remove them and replace them with a more useful error or avoid using an error at all